apps/frontend/src/components/global/upload_tabs/tenable/FileList.vue

```diff
@@ -0,0 +1,262 @@
+                tooltip.hover
+                title="Show 30 days cumulative scans"
+                @click="updateSearch"
+              />
+              <v-radio
+                label="60 Days"
+                :value="60"
+                tooltip.hover
+                title="Show 60 days cumulative scans"
+                @click="updateSearch"
+              />
+              <v-radio
+                label="90 Days"
+                :value="90"
+                tooltip.hover
+                title="Show 90 days cumulative scans"
+                @click="updateSearch"
+              />
+            </v-radio-group>
+          </v-sheet>
+        </v-col>
+      </v-row>
+    </v-container>
+
+    <v-spacer />
+
+    <div class="d-flex flex-column justify-content-between">
+      <v-data-table
+        v-model="selectedExecutions"
+        :headers="headers"
+        item-key="id"
+        :items="executions"
+        :items-per-page="5"
+        :loading="loading"
+        :show-sort-icons="true"
+        show-select
+      >
+        <template #no-data> No data. Try relaxing the scan options. </template>
+      </v-data-table>
+      <v-btn block class="card-outter" @click="loadResults">
+        Load Selected
+        <v-icon class="pl-2"> mdi-file-download</v-icon>
+      </v-btn>
+    </div>
+  </span>
+</template>
+
+<script lang="ts">
+import {InspecIntakeModule, FileLoadOptions} from '@/store/report_intake';
+import {SnackbarModule} from '@/store/snackbar';
+import {FileMetaData} from '@mitre/hdf-converters';
+import {AuthInfo, ScanResults, TenableUtil} from '@/utilities/tenable_util';
+import Vue from 'vue';
+import Component from 'vue-class-component';
+import {Prop} from 'vue-property-decorator';
+import {InspecDataModule} from '@/store/data_store';
+
+@Component({})
+export default class FileList extends Vue {
+  @Prop({type: Object, required: true}) readonly tenableConfig!: AuthInfo;
+
+  executions: Omit<FileMetaData, 'profile_sha256'>[] = [];
+  selectedExecutions: Omit<FileMetaData, 'profile_sha256'>[] = [];
+
+  loading = false;
+
+  scanDays = 0;
+
+  /** Table info */
+  headers = [
+    {
+      text: 'Scan ID',
+      value: 'id',
+      filterable: true,
+      align: 'start'
+    },
+    {
+      text: 'Name',
+      value: 'name',
+      align: 'start'
+    },
+    {
+      text: 'Details',
+      value: 'details',
+      align: 'start'
+    },
+    {
+      text: 'IPs',
+      value: 'scannedIPs'
+    },
+    {
+      text: 'Total Checks',
+      value: 'totalChecks'
+    },
+    {
+      text: 'Start Time',
+      value: 'startTime'
+    },
+    {
+      text: 'Finished Time',
+      value: 'finishTime'
+    },
+    {
+      text: 'Scan Status',
+      value: 'status'
+    }
+  ];
+
+  async updateSearch() {
+    this.loading = true;
+
+    // Get todays date
+    const dateNow = new Date();
+
+    // Convert date to epoch number value
+    const endTime = Number(dateNow.getTime().toString().substring(0, 10));
```

Comment:
what is the point of doing the substring?  looks like it's cutting off the last three digits of time *for now* so is the intent just to get the epoch time but at second resolution?  if so, then we can do something like: `Math.trunc(dateNow.getTime()/1000)` and avoid having to do the cast to string, latent bug string manipulation, and then recast back into number.

Reply:
yes, if you check epoch values are based on 10 digital numeric values, the getTime, returns eccessive digits (3) that when passed to the endpoint will not match the date time format on the server. The implemented solution is valid, no need to change it.